overrides/config/jei/worldSettings.cfg

```diff
@@ -0,0 +1,23 @@
+# Configuration file
+
+~CONFIG_VERSION: 0.1.0
+
+world-195213597 {
+    # Show the list of bookmarks next to open GUIs.
```

Comment:
Probably shouldn't be there.

Reply:
The file has been removed, since it only contains world specific data.